### ISSUE TITLE
pretraining epochs

### ISSUE THREAD

**Issue by BostonCrayfish** [deepmind/detcon#4]
Hi, how many epochs have you pretrained for the resnet50 checkpoint you provided? I noticed that the ema of batch norm parameters was counted to 625568, but I don't know the batch size you used in pretraining.

**Comment by olivierhenaff** [comment]
Hi, they've been trained for 1000 ImageNet epochs

**Comment by BostonCrayfish** [comment]
Got it! Thank you very much.
